WordPress/Classes/Extensions/UIColor+MurielColors.swift

```diff
@@ -0,0 +1,150 @@
+/// Generates the names of the named colors in the ColorPalette.xcasset
+enum MurielColorName: String, CustomStringConvertible {
+    // MARK: - Base colors
+    case blue
+    case celadon
+    case gray
+    case green
+    case hotBlue = "Hot-Blue"
+    case hotGreen = "Hot-Green"
+    case hotOrange = "Hot-Orange"
+    case hotPink = "Hot-Pink"
+    case hotPurple = "Hot-Purple"
+    case hotRed = "Hot-Red"
+    case hotYellow = "Hot-Yellow"
+    case jetpackGreen = "Jetpack-Green"
+    case orange
+    case pink
+    case purple
+    case red
+    case wooPurple = "Woo-Purple"
+    case yellow
+
+    var description: String {
+        return rawValue.capitalized
+    }
+}
+
+/// Value of a Muriel color's shade
+///
+/// Note: There are a finite number of acceptable values. Not just any Int works.
+///       Also, enum cases cannot begin with a number, thus the `shade` prefix.
+enum MurielColorShade: Int, CustomStringConvertible {
+    case shade0 = 0
+    case shade50 = 50
+    case shade100 = 100
+    case shade200 = 200
+    case shade300 = 300
+    case shade400 = 400
+    case shade500 = 500
+    case shade600 = 600
+    case shade700 = 700
+    case shade800 = 800
+    case shade900 = 900
+
+    var description: String {
+        return "\(rawValue)"
+    }
+}
+
+struct MurielColorIdentifier {
```

Comment:
How about just calling this `MurielColor` to make things a little more concise? Then it's made up of a name and shade.